src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,65 @@
 package mate.jdbc;
 
+import java.io.IOException;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.CreateDataBase;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+import mate.jdbc.service.impl.CreateDataBaseImpl;
+import mate.jdbc.util.FileUtils;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final String INIT_DB = "/init_db.sql";
+
     public static void main(String[] args) {
-        // test your code here
+        try {
+            CreateDataBase createDB = new CreateDataBaseImpl();
+            createDB.createdTable(FileUtils.readFile(INIT_DB));
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t create table for database");
+        }
```

Comment:
you didn't need to add all this in this task

Reply:
No, I thought it would be more interesting, when the application is launched, a database is formed